pkgs/development/libraries/qtfeedback/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, perl
+, qmake
+, qtbase
+, qtdeclarative
+}:
+
+mkDerivation rec {
+  pname = "qtfeedback";
+  version = "unstable-2018-09-03";
+
+  outputs = [ "out" "dev" ];
+
+  src = fetchFromGitHub {
+    owner = "qt";
+    repo = "qtfeedback";
+    rev = "a14bd0bb1373cde86e09e3619fb9dc70f34c71f2";
+    sha256 = "0kiiffvriagql1cark6g1qxy7l9c3q3s13cx3s2plbz19nlnikj7";
+  };
+
+  nativeBuildInputs = [
+    perl
+    qmake
+  ];
+
+  preConfigure = ''
+    perl ${qtbase.dev}/bin/syncqt.pl -version ${lib.getVersion qtbase}
```

Comment:
Qmake can run this automatically if you `leaveDotGit = true` in `src` (I think) or manually set `CONFIG+=git_build` in `qmakeFlags`, but it's currently broken (#76080). #118904 is supposed to fix that.

The `-version` value is then taken from the `.qmake.conf` file in the root of `src`, which is `MODULE_VERSION = 0.0.0`. I don't think this version reflects the Qt version it was built against, but the module's version for API compatibility? [Debian patches this to `5.0.0`](https://salsa.debian.org/qt-kde-team/qt/qtfeedback/-/blob/master/debian/patches/1001_revert_module_version.patch), [Alpine leaves it as `0.0.0`](https://pkgs.alpinelinux.org/contents?branch=edge&name=qt5-qtfeedback&arch=aarch64&repo=community). :man_shrugging: